crafting_pieces.xml

```diff
@@ -4670,10 +4670,10 @@
       <Material id="Iron6" count="3" />
     </Materials>
   </CraftingPiece>
-  <CraftingPiece id="crpg_aserai_noble_sword_3_t5_blade" name="{=xYgJN0Ja}Decorated Scimitar Blade" tier="5" piece_type="Blade" mesh="aserai_noble_blade_4" culture="Culture.aserai" length="87.4" weight="1.0">
+  <CraftingPiece id="crpg_aserai_noble_sword_3_t5_blade" name="{=xYgJN0Ja}Decorated Scimitar Blade" tier="5" piece_type="Blade" mesh="aserai_noble_blade_4" culture="Culture.aserai" length="96" weight=".75">

```

Comment:
lengths need to be edited in scale factor in weapon.xml. Do not edit length in crafting piece . Then length in crafting piece is set by the 3d modeller to fin the 3d model. Editing thje length here wil resulte in gaps between parts because of a difference between physical model and 3d model.